src/components/search-form/__tests__/SearchForm.test.js

```diff
@@ -67,6 +67,23 @@ describe('components/search-form/SearchForm', () => {
         sinon.assert.calledWithMatch(onSubmitSpy, 'cheese', event);
     });
 
+    test('should call onsubmit on search icon submit event', () => {
+        const onSubmitSpy = sinon.spy();
+        const event = {
+            target: {
+                elements: [
+                    {
+                        value: 'cheese',
+                    },
+                ],
+            },
+        };
+        const wrapper = mount(<SearchForm onSubmit={onSubmitSpy} placeholder="search" value="cheese" />);
+        const searchButton = wrapper.find('form').find('button.search-button');
+        searchButton.simulate('submit', event);
```

Comment:
Does this still pass if the button type was still `button`?

Reply:
It does, you're right. I will make sure to explicitly test for the "submit" type then